agent/command/s3_copy.go

```diff
@@ -204,6 +204,15 @@ func (c *s3copy) copyWithRetry(ctx context.Context,
 	for _, s3CopyFile := range c.S3CopyFiles {
 		timer.Reset(0)
 
+		if len(s3CopyFile.BuildVariants) > 0 && !utility.StringSliceContains(
+			s3CopyFile.BuildVariants, conf.BuildVariant.Name) {
+			continue
+		}
+
+		if ctx.Err() != nil {
+			return errors.New("s3copy operation received was canceled")
```

Comment:
I know this was just a copy paste but can we remove "received" so that this sentence makes sense?